10-tipi-cache.md

```diff
@@ -33,6 +33,41 @@ Whenever you launch a build tipi will produce a mirror of your sources either lo
 
 If you want to fully ignore data for the build you can add them to the `.tipiignore` file.
 
+## Downloading remote build results
+
+With tipi `v0.0.51` you can download build results directly onto your machine without downloading the full (potentially very big) build directory.
+
+Assuming we have a structure like this:
+```sh
+.
+├── src
+│  ├── string_to_json.cpp
+│  └── stringstream_mayham.cpp
+```
+
+We can now build the sources remotely like this:
+```sh
+tipi -t linux-cxx17 build .
+```
+
+Finally we can download the complete binaries like this:
+```sh
+tipi -t linux-cxx17 download "./build/linux-cxx17/bin/src/string*"
```

Comment:
I would also document just downloading a single file at first

Reply:
Should we document the variant without the `build/plattform/bin/` as well?

Reply:
No it's fine this way thanks !